core/unit_test/CMakeLists.txt

```diff
@@ -87,7 +87,12 @@ macro(Cabana_add_tests)
   endforeach()
 endmacro()
 
-Cabana_add_tests(NAMES AoSoA Slice DeepCopy Tuple Sort LinkedCellList NeighborList Parallel)
+# FIXME NeighborList unit test does not compile with the HIP backend enabled yet
+if(Kokkos_ENABLE_HIP)
+  Cabana_add_tests(NAMES AoSoA Slice DeepCopy Tuple Sort LinkedCellList Parallel)
+else()
+  Cabana_add_tests(NAMES AoSoA Slice DeepCopy Tuple Sort LinkedCellList NeighborList Parallel)
```

Comment:
No `NeighborList` because of hierarchical parallelism right?

Reply:
Commented below - maybe I'm misunderstanding